modules/observe/scorecards/proc-authenticating-and-managing-scorecard-plugins.adoc

```diff
@@ -0,0 +1,31 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="proc-authenticating-and-managing-scorecard-plugins_{context}"]
+= Authenticating and managing Scorecard plugins to control who sees the metrics
+
+As an administrator, you can secure the connection to your metric providers by managing user access to the Scorecards using the Role-Based Access Control (RBAC) permissions. For users to view the Scorecard metrics, you must provide users with read access to the Scorecard metrics. You can configure these permissions either by modifying your RBAC csv file directly or by using the RBAC Web UI.
+
+.Prerequisite
+ * You have {authorization-book-link}#enabling-and-giving-access-to-rbac[enabled RBAC, have a policy administrator role in Developer Hub, and have added scorecard to plugins with permission].
+
+.Procedure
+. Set the required RBAC permissions using any of the following methods:
```

Comment:
we can't have a single step procedure. So, i would suggest to change this to a normal sentence rather than giving it a numbered list. 
```suggestion
Use any of the following methods to set the required RBAC permissions:
```